squidDrone/src/imu/inertial_measurement.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef SRC_IM_HPP_
```

Comment:
SRC_IM_HPP is not very unique. Please choose a more unique keyword to prevent inclusion conflicts

Reply:
Renaming done